go/pkg/rexec/rexec.go

```diff
@@ -127,6 +127,32 @@ func (ec *Context) downloadOutputs(execRoot string) *command.Result {
 	return command.NewResultFromExitCode((int)(ec.resPb.ExitCode))
 }
 
+// HighAPIVersionNewerThanOrEqualTo returns whether the latest version reported
+// as supported in ServerCapabilities matches or is more recent than a
+// reference major/minor version.
+func HighAPIVersionNewerThanOrEqualTo(serverCapabilities *repb.ServerCapabilities, major int32, minor int32) bool {
```

Comment:
Please make this private. Later, I'll move it out into the client, probably be something like client.serverSupports(apiVersion)->supportStatus

Reply:
Sounds good. That approach would be much nicer!